shared/team-building/service-tab-bar.desktop.tsx

```diff
@@ -85,24 +165,65 @@ const styles = Styles.styleSheetCreate({
     borderColor: Styles.globalColors.black_10,
     height: 2,
   },
+  label: {
+    marginTop: Styles.globalMargins.xtiny,
+    minWidth: 64,
+  },
+  moreNetworks1: {
+    flex: 1,
+    paddingBottom: Styles.globalMargins.tiny,
+    paddingLeft: Styles.globalMargins.xsmall,
+    paddingRight: Styles.globalMargins.xsmall,
+    paddingTop: Styles.globalMargins.tiny,
+  },
+  moreNetworks2: {
+    alignItems: 'center',
+    display: 'flex',
+    flexDirection: 'row',
+    height: '100%',
+    justifyContent: 'center',
+    maxHeight: '100%',
+    maxWidth: '100%',
+    width: '100%',
+  },
+  moreNetworks3: {
+    alignItems: 'center',
+    borderColor: Styles.globalColors.black_20,
+    borderRadius: 4,
+    borderStyle: 'solid',
+    borderWidth: 1,
+    display: 'flex',
+    flexDirection: 'row',
+    height: '100%',
+    justifyContent: 'center',
+    maxHeight: '100%',
+    maxWidth: '100%',
+    width: '100%',
+  },
+  moreText: {
+    color: Styles.globalColors.black_50,
+  },
   pendingIcon: {height: 10, width: 10},
-  resultCount: {},
   serviceIcon: {
     marginRight: Styles.globalMargins.xtiny,
   },
+  serviceIconBox: {
+    marginTop: 14,
```

Comment:
I'm always a little hesitant to make margins that aren't part of `globalMargins`. Does this need to be 12 or can we round it to be `xsmall` or `small`?

Reply:
14 matches the design exactly